### DISCUSSION TITLE
osu!wiki links aren't handled properly by in-game website overlay

### DISCUSSION THREAD

**Discussion by venix12** [ppy/osu#17084] [Ideas] [answered]
URLs used by osu-web aren't compatible with those used in lazer client which causes them to error when opened from the chat.

![](https://i.imgur.com/uEmAbrh.gif)

The URLs without specified locale (https://osu.ppy.sh/wiki/Beatmap_Spotlights/Seasons/2022_Winter in this case) work fine, yet they are autmatically redirected to the one with locale specified (https://osu.ppy.sh/wiki/en/Beatmap_Spotlights/Seasons/2022_Winter) on the web side of things so it might come as counterintuitive when copy-pasted to the chat.

**Comment by bdach** [comment, marked as answer]
#17085
